### ISSUE TITLE
Invalid worker doesn't show a syntax error

### ISSUE THREAD

**Issue by yasassri** [ballerinalang/composer#3819]
![screenshot from 2017-10-17 00-13-31](https://user-images.githubusercontent.com/7681361/31629032-1364fa26-b2d0-11e7-8b65-97b1c3d205cc.png)

**Comment by manuranga** [comment]
@hasithaa @lafernando 
this is cause by because run-time throws a NPE
```
java.lang.NullPointerException
	at org.wso2.ballerinalang.compiler.semantics.analyzer.CodeAnalyzer$WorkerActionSystem.addWorkerAction(CodeAnalyzer.java:833)
	at org.wso2.ballerinalang.compiler.semantics.analyzer.CodeAnalyzer.visit(CodeAnalyzer.java:583)
	at org.wso2.ballerinalang.compiler.tree.statements.BLangWorkerSend.accept(BLangWorkerSend.java:75)
	at org.wso2.ballerinalang.compiler.semantics.analyzer.CodeAnalyzer.lambda$visit$4(CodeAnalyzer.java:326)
	at java.util.ArrayList.forEach(ArrayList.java:1249)
	at org.wso2.ballerinalang.compiler.semantics.analyzer.CodeAnalyzer.visit(CodeAnalyzer.java:325)
```

We'll try to catch and show an error, should we fix in run time as well?

**Comment by lafernando** [comment]
Hi Manu,

The runtime is fixed with this [1].

[1] https://github.com/ballerinalang/ballerina/pull/3653

Cheers,
Anjana.

**Comment by manuranga** [comment]
Fixed in composer at #3818
